pylint/checkers/base/comparison_checker.py

```diff
@@ -157,7 +157,7 @@ def _is_float_nan(node: nodes.NodeNG) -> bool:
         def _is_numpy_nan(node: nodes.NodeNG) -> bool:
             match node:
                 case nodes.Attribute(attrname="NaN", expr=nodes.Name(name=name)):
-                    return name in {"numpy", "nmp", "np"}
+                    return name in {"numpy", "np"}
```

Comment:
It was noted during one of the PRs that `nmp` isn't really used anymore. IMO `numpy` and `np` should be enough. Happy to revert that though.

Reply:
- https://github.com/search?q=%22import+numpy+as+nmp%22&type=code
- https://github.com/search?q=%22as+nmp%22+language%3APython&type=code&l=Python

576 occurrence on the whole of github, possibly 16 import mistakenly identified as numpy (but they probably don't have a NaN ?)

Reply:
Not sure if it's worth it, but we should probably not do it in an underhanded way in an unrelated MR.